src/Core/src/Platform/iOS/ImageViewExtensions.cs

```diff
@@ -21,24 +22,41 @@ public static void UpdateAspect(this UIImageView imageView, IImage image)
 			imageView.ClipsToBounds = imageView.ContentMode == UIViewContentMode.ScaleAspectFill || imageView.ContentMode == UIViewContentMode.Center;
 		}
 
-		public static void UpdateIsAnimationPlaying(this UIImageView imageView, IImageSourcePart image)
+		public static async Task UpdateIsAnimationPlaying(this UIImageView imageView, IImageSourcePart image)
```

Comment:
This is a breaking change, so maybe this should be a new method that is `UpdateIsAnimationPlayingAsync` and then then old method can use FireAndForget:

```cs
public static void UpdateIsAnimationPlaying(this UIImageView imageView, IImageSourcePart image) =>
    imageView.UpdateIsAnimationPlayingAsync(image).FireAndForget(<something-here>);
```

Reply:
Good point, done!